### PR TITLE
Tab refetch policy

### PR DESCRIPTION
**Pull request by asleepingpanda**
### 1. What does this PR do, or why is it needed?
When the app moves _back_ into a foreground, active state, refetch data for the Feature Feeds

### CODE REVIEW

File: src/client/index.js [hunk 1]
```diff
@@ -44,33 +44,18 @@ export const client = new ApolloClient({
...
-  //   },
-  // },
+  typePolicies: {
+    Query: {
+      fields: {
+        PeopleConnection: {
+          merge(existing = [], incoming) {
+            const mappedIncoming = incoming.map(({ edges }) => edges.cursor);
+            return { ...existing, ...mappedIncoming };
+          },
```
Comment (rkdavidson) [inline src/client/index.js:54]: This looks unrelated to the refetch policy change — intentional?
Is there additional context here?

**Review by rkdavidson — APPROVED** [review]
Haven't tested locally (in thick of Fastlane setup so local environment is fragile), but code looks ok.

Was there a specific ask for this feature?
Is there risk that we'll be increasing load on the API unnecessarily?

I'm skeptical of this change since hypothetically people could open the app, pop over to messages to respond to a text for a second, then come back and have to re-fetch a fairly hefty chunk of data that they already have.

It feels like it errs on the "always show up to date content" at expense of performance/API load.

If that's acceptable or this fixes some bug, I approve.

File: src/client/index.js (hunk 1 above)
Comment (asleepingpanda) [reply src/client/index.js:54]: That code was suppose to be uncommented a while ago, so I'm just resolving that issue here cause I noticed it.

**Comment by asleepingpanda** [comment]
Yeah so there's an ongoing problem that keeps getting brought up where people are seeing old data because they are not pulling to refresh. The resolution was just to listen to app state and refresh your feeds when you bring the app back into a focused state.
